.changeset/fruity-moles-fry.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/container-definitions": major
+"@fluidframework/runtime-definitions": major
+---
+
+IContainer's and IDataStore's IFluidRouter capabilities are deprecated
+
+While the `request` function taking an arbitrary URL and headers is deprecated, an overload taking only `{ url: '/' } is not,
+for back-compat purposes during the migration from the request pattern to using entryPoint.
```

Comment:
Oops I didn't give alternative instructions in here.  Will update, and also update Removing-IFluidRouter.md